server/courselist.rb

```diff
@@ -219,21 +221,23 @@ def self.configureStatic
   ## make sure logging is available
   configure :test do
 
-    #set :logging, Logger::DEBUG
-    set :logging, Logger::INFO
+    set :logging, Logger::DEBUG
+    #set :logging, Logger::INFO
 
     #configureLogging
 
     ## look for the UI files in a parallel directory.
     ## this may not be necessary.
     configureStatic
-    set :logging, Logger::INFO
+    #set :logging, Logger::INFO
+    set :logging, Logger::DEBUG
```

Comment:
Yes. It should default to INFO.  The Ruby way to change it will be to use an environment variable.